reinforcement_learning/rlclientlib/logger/event_batcher.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
```

Comment:
event_batcher seems to have overlapping batching logic with other parts of code.  Also seems to violate the spirit of high water mark.